resources/fetchers/kube_provider.go

```diff
@@ -80,6 +80,8 @@ func (r K8sResource) GetMetadata() fetching.ResourceMetadata {
 	}
 }
 
+func (r K8sResource) GetElasticCommonData() any { return nil }
```

Comment:
shouldn't this sit in the kube_fetcher?

Reply:
Seems fine to me to reside next to the rest of the `Resource interface`
```
type Resource interface {
	GetMetadata() ResourceMetadata
	GetData() any
	GetElasticCommonData() any
}
```

If you feel this is not the place we can move them all.